tests/settings.py

```diff
@@ -21,14 +21,16 @@
     # local sqlite database file
     DATABASES['default'] = {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': INSTALL_DIR + 'db.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
```

Comment:
Ah, I just read your comment re: Linux installation. If you use `pyenv` or `virtualenv`, I don't think this would be a problem, but lets go with this for increased robustness